tests/unit/test_pull_request.py

```diff
@@ -115,13 +140,21 @@ def test_repository_client_create_branch(
     upstream_repository: Repo,
 ):
     """
-    arrange: given RepositoryClient and newly created files in repo directory
+    arrange: given RepositoryClient and newly created files in repo and repo/docs directory
     act: when _create_branch is called
-    assert: a new branch is successfully created upstream.
-    """
-    testfile = "testfile.txt"
-    testfile_content = "test"
-    (repository_path / testfile).write_text(testfile_content)
+    assert: a new branch is successfully created upstream with only the files in the repo/docs
```

Comment:
Shoudn't it be `in the repo and repo/docs directories` ?

Reply:
Only the files in the `docs` directory within the `repo` should actually be part of the migration branch, if any other files are accidentally created, they should be ignored. This mainly helps for the e2e test which creates some fixtures which ended up in the migration PR before